packages/web/src/services/solana/solana.ts

```diff
@@ -9,16 +9,14 @@ import {
 import { PublicKey, Transaction, Keypair } from '@solana/web3.js'
 
 import { getLibs } from 'services/audius-libs'
+import { audiusSdk } from 'services/audius-sdk'
 
 export const ROOT_ACCOUNT_SIZE = 0 // Root account takes 0 bytes, but still pays rent!
 export const TRANSACTION_FEE_FALLBACK = 10000
 
-/**
- * Gets the solana connection from libs.
- */
 export const getSolanaConnection = async () => {
-  const libs = await getLibs()
-  return libs.solanaWeb3Manager!.getConnection()
+  const sdk = await audiusSdk()
+  return sdk.services.claimableTokensClient.connection
```

Comment:
I think I understood the *Client paradigm to be that you invoked a client for each program you interacted with. If that's true, I'm curious why the ClaimableTokens client is the one that gets used as the connection source for all solana interactions?

Reply:
Yeah this sucks and I want to rewrite it. Initially I just had this initialize a new connection from the `env`. There's no "root" connection to pull out of SDK because each client is separately initted. I think the solve here is some "root" `SolanaClient` in SDK that each *Client is composed of rather than inheriting the `BaseSolanaClient`. Will take a stab at refactoring that.

Reply:
Okay no worries for this PR. I just wanted to make sure I understood the intended design. I can totally see that since we need a root connection and we don't have a `SolanaWeb3Manager` in SDK, this is a compromise solution that gets us access to a connection when we need it.

Reply:
https://github.com/AudiusProject/audius-protocol/pull/9441 👀